integration-tests/contract-batch.spec.ts

```diff
@@ -5,18 +5,17 @@ import { MANAGER_LAMBDA, OpKind } from '@taquito/taquito';
 
 CONFIGS().forEach(({ lib, rpc, setup, knownBaker, knownContract, createAddress }) => {
     const Tezos = lib;
-    const test = require('jest-retries');
     describe(`Test contract.batch using: ${rpc}`, () => {
         beforeEach(async (done) => {
             await setup();
             done();
         });
-        test('Simple transfers with origination (where the code in JSON Michelson format)', 2, async (done: () => void) => {
+        test('Simple transfers with origination (where the code in JSON Michelson format)', async (done: () => void) => {
```

Comment:
I think that `: () => void` was added for jest-retries and can be removed

`async (done)` instead of `async (done: () => void)`

Reply:
I have made this fix in several places where it is needed.